src/eventsub/mod.rs

```diff
@@ -166,6 +166,42 @@ impl<E: EventSubscription + Clone> Message<E> {
 
 impl<E: EventSubscription> Payload<E> {
     /// Parse string slice as a [`Payload`], this will assume your string is from an eventsub message with type `notification`
+    ///
+    /// # Examples
```

Comment:
this should mention that it should only be an object with `subscription` and `event`